commit 1b1934dbbdcf9aa2d507932ff488cec47999cf3f
Merge: bce3b5d6764b ead8467f96d6
Author: Linus Torvalds <torvalds@linux-foundation.org>
Date:   Wed Jan 17 11:49:11 2024 -0800

    Merge tag 'docs-6.8-2' of git://git.lwn.net/linux
    
    Pull documentation fixes from Jonathan Corbet:
     "A handful of late-arriving documentation fixes"
    
    * tag 'docs-6.8-2' of git://git.lwn.net/linux:
      docs, kprobes: Add loongarch as supported architecture
      docs, kprobes: Update email address of Masami Hiramatsu
      docs: admin-guide: hw_random: update rng-tools website
      Documentation/core-api: fix spelling mistake in workqueue
      docs: kernel_feat.py: fix potential command injection
      Documentation: constrain alabaster package to older versions

diff --cc Documentation/sphinx/requirements.txt
index a8a1aff6445e,6b0a981dcb2c..5d47ed443949
--- a/Documentation/sphinx/requirements.txt
+++ b/Documentation/sphinx/requirements.txt
@@@ -1,4 -1,5 +1,6 @@@
  # jinja2>=3.1 is not compatible with Sphinx<4.0
  jinja2<3.1
+ # alabaster>=0.7.14 is not compatible with Sphinx<=3.3
+ alabaster<0.7.14
  Sphinx==2.4.4
 +pyyaml